web/src/hooks/api/config/storage.ts

```diff
@@ -20,24 +20,23 @@
  * find current contact information at www.suse.com.
  */
 
-import { useSuspenseQuery, useQuery } from "@tanstack/react-query";
-import { QueryHookOptions } from "~/types/queries";
-import { extendedConfigQuery } from "~/hooks/api";
+import { useSuspenseQuery } from "@tanstack/react-query";
+import { extendedConfigQuery } from "~/hooks/api/config";
 import { putConfig, Response } from "~/api";
 import { Config } from "~/api/config";
 
-const resetConfig = (data: Config | null): Config => (!data ? {} : { ...data, storage: null });
+const removeStorageConfig = (data: Config | null): Config =>
+  !data ? {} : { ...data, storage: null };
 
-type ResetConfigFn = () => Response;
+type ResetFn = () => Response;
 
-function useResetConfig(options?: QueryHookOptions): ResetConfigFn {
-  const func = options?.suspense ? useSuspenseQuery : useQuery;
-  const { data } = func({
-    ...extendedConfigQuery(),
-    select: resetConfig,
+function useReset(): ResetFn {
+  const { data } = useSuspenseQuery({
+    ...extendedConfigQuery,
+    select: removeStorageConfig,
   });
   return () => putConfig(data);
 }
 
-export { useResetConfig };
-export type { ResetConfigFn };
+export { useReset };
+export type { ResetFn as ResetConfigFn };
```

Comment:
what is reason for this rename? I would not rename types unless there is good reason, as it make grepping source codes for its definition harder.